apps/dashboard/src/components/workflow-editor/configure-workflow.tsx

```diff
@@ -1,215 +1,14 @@
-                      disabled={isReadOnly}
-                      onFocus={() => setIsBlurred(false)}
-                      onBlur={() => setIsBlurred(true)}
-                    />
-                  </InputField>
-                </FormControl>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
-          <FormField
-            control={control}
-            name="workflowId"
-            defaultValue=""
-            render={({ field }) => (
-              <FormItem>
-                <FormLabel>Identifier</FormLabel>
-                <FormControl>
-                  <InputField className="flex overflow-hidden pr-0">
-                    <Input placeholder="Untitled" className="cursor-default" {...field} readOnly />
-                    <CopyButton size="input-right" valueToCopy={field.value} />
-                  </InputField>
-                </FormControl>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
-          <FormField
-            control={control}
-            name="description"
-            render={({ field }) => (
-              <FormItem>
-                <FormLabel>Description</FormLabel>
-                <FormControl>
-                  <Textarea
-                    className="min-h-36"
-                    placeholder="Description of what this workflow does"
-                    {...field}
-                    maxLength={MAX_DESCRIPTION_LENGTH}
-                    disabled={isReadOnly}
-                  />
-                </FormControl>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
-          <FormField
-            control={control}
-            name="tags"
-            render={({ field }) => (
-              <FormItem className="group" tabIndex={-1}>
-                <div className="flex items-center gap-1">
-                  <FormLabel>Tags</FormLabel>
-                </div>
-                <FormControl className="text-xs text-neutral-600">
-                  <TagInput
-                    {...field}
-                    disabled={isReadOnly}
-                    value={field.value ?? []}
-                    suggestions={tagsQuery.data?.data.map((tag) => tag.name) || []}
-                  />
-                </FormControl>
-                <FormMessage />
-              </FormItem>
-            )}
-          />
-        </SidebarContent>
-        <Separator />
-        {/* <SidebarContent size="lg">
-        <Button variant="outline" className="flex w-full justify-start gap-1.5 text-xs font-medium" type="button">
-          <RiSettingsLine className="h-4 w-4 text-neutral-600" />
-          Configure channel preferences <RiArrowRightSLine className="ml-auto h-4 w-4 text-neutral-600" />
-        </Button>
-      </SidebarContent>
-      <Separator /> */}
-      </motion.div>
-    </>
-  );
+  return <ConfigureWorkflowForm workflow={workflow} debouncedUpdate={debouncedUpdate} patch={patch} />;
```

Comment:
I don't think we need both the `debouncedUpdate` and the patch. I'd pass a `save` prop.

Reply:
patch is used for active field only. update is instant PUT. debouncedUpdate is debounced PUT